tests/model_monitoring/db/tsdb/timescaledb/test_timescaledb_connector.py

```diff
@@ -0,0 +1,232 @@
+                        connector.delete_tsdb_resources()
+
+    def test_auto_create_database_without_system_id_raises_error(
+        self, project_name, profile
+    ):
+        """Test that auto_create_database=True without system_id raises error."""
+        import mlrun
+        from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_connector import (
+            TimescaleDBConnector,
+        )
+
+        with self._config_context(system_id="", auto_create=True):
+            # Should raise error - no cleanup needed since connector creation fails
+            with pytest.raises(mlrun.errors.MLRunInvalidArgumentError) as exc_info:
+                TimescaleDBConnector(
+                    project=project_name,
+                    profile=profile,
+                )
+
+            assert "system_id is not set" in str(exc_info.value)
+
+    def test_auto_create_database_idempotent(
+        self, project_name, test_system_id, profile, admin_connection
+    ):
+        """Test that creating database multiple times is idempotent."""
+        from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_connection import (
+            Statement,
+        )
+        from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_connector import (
+            TimescaleDBConnector,
+        )
+
+        expected_db_name = f"mlrun_mm_{test_system_id}"
+        connector1 = None
+        connector2 = None
+
+        with self._config_context(system_id=test_system_id, auto_create=True):
+            try:
+                # Create first connector
+                connector1 = TimescaleDBConnector(
+                    project=project_name,
+                    profile=profile,
+                )
+                connector1.create_tables()
+
+                # Create second connector with same system_id - should not fail
+                connector2 = TimescaleDBConnector(
+                    project=f"{project_name}-2",
+                    profile=profile,
+                )
+                connector2.create_tables()
+
+                # Both should work - verify database exists
+                result = admin_connection.run(
+                    query=Statement(
+                        sql="SELECT 1 FROM pg_database WHERE datname = %s",
+                        parameters=(expected_db_name,),
+                    )
+                )
+                assert result.data, f"Database {expected_db_name} should exist"
+
+            finally:
+                # Cleanup resources - ignore errors
+                if connector1:
+                    with contextlib.suppress(Exception):
+                        connector1.delete_tsdb_resources()
+                if connector2:
+                    with contextlib.suppress(Exception):
+                        connector2.delete_tsdb_resources()
+                # Force drop database
+                self._force_drop_database(admin_connection, expected_db_name)
```

Comment:
No need to repeat the method name in a comment.
```suggestion
                self._force_drop_database(admin_connection, expected_db_name)
```

Reply:
Removed redundant comments.